### ISSUE TITLE
Question about the agent performance

### ISSUE THREAD

**Issue by wwwangzhch** [wassname/rl-portfolio-management#24]
Hi,
I am a pytorch user, so I studied the pytorch version of your codes. Although the reward of the training data is increasing, it fluctuates between positive and negative in the test dataset. It  seem the agent can't be able to get a good return on the test data.
If I miunderstood anything plz let me know, thank you!
